lib/languages.ts

```diff
@@ -27,28 +27,21 @@ import path from 'path';
 import fs from 'fs-extra';
 import _ from 'underscore';
 
-/***
```

Comment:
Thanks :) unless we make a concerted effort, these types of comments rot pretty quickly; and now we have some info in the type system istelf!